src/operator/elemwise_op_common.h

```diff
@@ -205,16 +261,22 @@ struct ElemwiseGradUseNone {
   }
 };
 
-struct CloneGradient {
```

Comment:
Wasn't it used here? 
https://github.com/apache/incubator-mxnet/blob/0b1363116c84dcefa751a925749b2da04c3f2614/src/operator/tensor/elemwise_binary_op_basic.cc#L40